### DISCUSSION TITLE
How to apply forces to particular points?

### DISCUSSION THREAD

**Discussion by lattice0** [bevyengine/bevy#5726] [Q&A] [answered]
Following my X shape test https://github.com/bevyengine/bevy/discussions/5698, where I use an X shape GLTF Blender mesh as a Collision shape, I now want to apply forces to one of the ends of the X shape. I couldn't find anything about forces, I think the tutorials mainly to translations on the object.

I think first I need to learn how to mark the points where I'll apply the force. I don't know if it's possible to do it in Blender. 

Then I need to learn how to load these points in bevy and apply the force on one of these points

**Comment by mockersf** [comment, marked as answer]
Bevy itself doesn't yet have a physics engine. For now, you'll need to pick one available in https://bevyengine.org/assets/#physics and see how to use it to apply forces
